Library/DataSource/UICollectionView-Extensions.swift

```diff
@@ -5,6 +5,10 @@ public extension UICollectionView {
     register(cellClass, forCellWithReuseIdentifier: cellClass.description())
   }
 
+  public func register<T: ValueCell>(_ cellClass: T.Type) {
```

Comment:
This is a helper that is compatible with `ValueCellDataSource`, which has a default implementation for `T.defaultReusableId` that's slightly different from the above `cellClass.description()`.

Reply:
I think ideally all of our cells should conform to `ValueCell`, so there can be consistency in how we register and configure all our cells.